source/codegen/metadata/nixnetsocket/custom_proto.mako

```diff
@@ -74,17 +74,17 @@ message AddrInfoHint {
 }
 
 message Linger {
-  int32 l_onoff = 1;
-  int32 l_linger = 2;
+  int32 onoff = 1;
+  int32 linger = 2;
 }
 
 message IPMReq {
-  InAddr imr_multiaddr = 1;
+  InAddr multiaddr = 1;
   InAddr imr_interface = 2;
```

Comment:
Note: I'm leaving the `interface` fields with the redundant prefix. `interface` is a compiler-specific keyword in MSVC (NOT a C/C++ keyword). So, on my setup, any file that includes windows.h and a `pb.h` file with an interface field in it will fail to compile. It is possible to disable this (supposedly). But I was unable to get it to work locally, and I'm "at least average" at configuring msvc options and cmake. So I think we can say this is a real issue for any windows clients and could easily affect other languages. Although there's no specific standard in protobuf for avoiding any language-specific reserved keywords, the actual behavior in different languages is often pretty bad. So it's reasonable to avoid when possible.

FYI @doshirohan I am planning to make the same change for xnet and I'll ad you to the review. Sorry for the confusion.